relay-server/src/actors/project.rs

```diff
@@ -601,6 +606,7 @@ impl Project {
     ///
     /// The buckets will be keyed underneath this project key.
     pub fn merge_buckets(&mut self, buckets: Vec<Bucket>) {
+        // TODO: rate limits
```

Comment:
We will add a call to `BucketLimiter::enforce_limits()` to `metrics_allowed()` in a follow-up PR.